src/Containers/LoginPage/LoginPage.js

```diff
@@ -0,0 +1,215 @@
+                email:email,
+                password:password
+            })
+    }
+
+    const changePasswordSubmit=async (event)=>{
+        event.preventDefault();
+        validateEmail();
+        if(password===confirmPassword){
+            changePassword({
+                email:email,
+                password:password,
+                confirmPassword:confirmPassword
+            })
+        }else{
+            warningToast("Passwords do not match")
+        }
+    }
+
+    const pageToRender=()=>{
+        if(currentPage==="SIGNUP_PAGE"){
+            return(
+                <>
+                    <h1>
+                        Sign Up:
+                    </h1>
+                    <form 
+                        className={classes["signup-container"]}
+                        onSubmit={signUpSubmit}
+                    >
+                        <div>
+                            <input type="text" 
+                                className={classes["textbox"]} 
+                                placeholder="Username"
+                                required
+                                value={userName}
+                                onChange={event=>setUserName(event.target.value)}
+                            />
+                            {!userNameValid&&<p className={classes["error-text"]}>Please enter a valid user name</p>}
+                        </div>
+                        <div>
+                            <input type="email" 
+                                className={classes["textbox"]}
+                                placeholder="Email"
+                                required
+                                value={email}
+                                onChange={event=>setEmail(event.target.value)}
+                            />
+                            {!emailValid&&<p className={classes["error-text"]}>Please enter a valid email</p>}
+                        </div>
+                        <input 
+                            type="password" 
+                            className={classes["textbox"]} 
+                            placeholder="Password"
+                            required
+                            value={password}
+                            onChange={event=>setPassword(event.target.value)}
+                        />
+                        <button 
+                            type="submit"
+                            className={`${classes["button-solid"]} ${classes["button-primary"]}`}    
+                        >
+                            Sign up!
+                        </button>
+                    </form>
+                </>
+            )
+        }else if(currentPage==="SIGNIN_PAGE"){
+            return(
+                <>
+                    <h1>
+                        Sign In:
+                    </h1>
+                    <form 
+                        className={classes["signup-container"]}
+                        onSubmit={signInSubmit}
```

Comment:
Here we can use `switch(currentPage)` to refactor.

Reply:
Yeah I'll make the changes